geth/transactions/txqueue_manager_test.go

```diff
@@ -63,25 +63,31 @@ func (s *TxQueueTestSuite) TearDownTest() {
 	s.client.Close()
 }
 
-func (s *TxQueueTestSuite) setupTransactionPoolAPI(tx *common.QueuedTx, config *params.NodeConfig, account *common.SelectedExtKey, nonce *hexutil.Uint64, gas *hexutil.Big, txErr error) {
+var (
+	testGas      = (*hexutil.Big)(big.NewInt(defaultGas + 1))
+	testGasPrice = (*hexutil.Big)(big.NewInt(10))
+	testNonce    = hexutil.Uint64(10)
```

Comment:
i will be using non-default nonce here https://github.com/status-im/status-go/pull/538 , so it is not very good idea. but you can leave it if you want. i can rebase

Reply:
Oh, okay. I'll just revert changes for nonce then.